src/Credentials/ServiceAccountCredentials.php

```diff
@@ -67,6 +68,11 @@ class ServiceAccountCredentials extends CredentialsLoader implements SignBlobInt
      */
     protected $auth;
 
+    /**
+     * The quota project associated with the JSON credentials
```

Comment:
nit: (applies to other credential types as well)

```suggestion
     * The quota project associated with the JSON credentials
     *
     * @var string
```

Reply:
> nit: (applies to other credential types as well)

This is true, but in the context of `ServiceAccountCredentials`, it will always be from JSON credentials. Would it be better as `The quota project associated with the service account credentials`?